server/grpc/newcore/core.go

```diff
@@ -0,0 +1,27 @@
+package newcore
```

Comment:
Can you move this file to `server/grpc/api/server.go` and the create function in `server/grpc/api/service.go` and renamed it `CreateService`

Reply:
done